### PR TITLE
[HOTFIX] relative import in Auth.ts

### PR DESCRIPTION
**Pull request by xbill82**
Fixes this [build error](https://app.netlify.com/sites/kuzzle-admin-console/deploys/60099111179c6800077bab7a) (seen on the Amin Console Netlify build). Webpack looks for a package called `src/types/RequestPayload` since the path doesn't start with `.`.

### CODE REVIEW

File: src/controllers/Auth.ts [hunk 1]
```diff
@@ -482,7 +482,7 @@ export class AuthController extends BaseController {
    * @param options Additional options
    *    - `queuable` If true, queues the request during downtime, until connected to Kuzzle again
    */
-  validateMyCredentials (
+  validateMyCredentials(
```
Comment (Aschen) [inline src/controllers/Auth.ts:485]:
```suggestion
  validateMyCredentials (
```

File: src/controllers/Auth.ts [hunk 2]
```diff
@@ -458,7 +458,7 @@ export class AuthController extends BaseController {
    *
    * @returns Currently logged User
    */
-  updateSelf (
+  updateSelf(
```
Comment (Aschen) [inline src/controllers/Auth.ts:461]:
```suggestion
  updateSelf (
```

File: src/controllers/Auth.ts [hunk 3]
```diff
@@ -433,7 +433,7 @@ export class AuthController extends BaseController {
    * @returns An object representing the updated credentials.
    *    The content depends on the authentication strategy
    */
-  updateMyCredentials (
+  updateMyCredentials(
```
Comment (Aschen) [inline src/controllers/Auth.ts:436]:
```suggestion
  updateMyCredentials (
```

File: src/controllers/Auth.ts [hunk 4]
```diff
@@ -411,7 +411,7 @@ export class AuthController extends BaseController {
    *
    * @see https://docs.kuzzle.io/sdk/js/7/controllers/auth/logout
    */
-  logout (): Promise<void> {
+  logout(): Promise<void> {
```
Comment (Aschen) [inline src/controllers/Auth.ts:414]:
```suggestion
  logout (): Promise<void> {
```

File: src/controllers/Auth.ts [hunk 5]
```diff
@@ -380,10 +380,10 @@ export class AuthController extends BaseController {
    *
    * @returns The encrypted JSON Web Token
    */
-  login (
+  login(
```
Comment (Aschen) [inline src/controllers/Auth.ts:383]:
```suggestion
  login (
```

File: src/controllers/Auth.ts [hunk 6]
```diff
@@ -361,7 +361,7 @@ export class AuthController extends BaseController {
    *
    * @returns An array of available strategies names
    */
-  getStrategies (options: { queuable?: boolean } = {}): Promise<Array<string>> {
+  getStrategies(options: { queuable?: boolean } = {}): Promise<Array<string>> {
```
Comment (Aschen) [inline src/controllers/Auth.ts:364]:
```suggestion
  getStrategies (options: { queuable?: boolean } = {}): Promise<Array<string>> {
```

File: src/controllers/Auth.ts [hunk 7]
```diff
@@ -321,7 +321,7 @@ export class AuthController extends BaseController {
    *
    * @returns An array containing user rights objects
    */
-  getMyRights (
+  getMyRights(
```
Comment (Aschen) [inline src/controllers/Auth.ts:324]:
```suggestion
  getMyRights (
```

File: src/controllers/Auth.ts [hunk 8]
```diff
@@ -278,7 +278,7 @@ export class AuthController extends BaseController {
    *
    * @returns Currently logged User
    */
-  getCurrentUser (options: { queuable?: boolean } = {}): Promise<User> {
+  getCurrentUser(options: { queuable?: boolean } = {}): Promise<User> {
```
Comment (Aschen) [inline src/controllers/Auth.ts:281]:
```suggestion
  getCurrentUser (options: { queuable?: boolean } = {}): Promise<User> {
```

File: src/controllers/Auth.ts [hunk 9]
```diff
@@ -257,7 +257,7 @@ export class AuthController extends BaseController {
    * @param options Additional options
    *    - `queuable` If true, queues the request during downtime, until connected to Kuzzle again
    */
-  deleteMyCredentials (
+  deleteMyCredentials(
```
Comment (Aschen) [inline src/controllers/Auth.ts:260]:
```suggestion
  deleteMyCredentials (
```

File: src/controllers/Auth.ts [hunk 10]
```diff
@@ -237,7 +237,7 @@ export class AuthController extends BaseController {
    *
    * @returns A boolean indicating if the credentials exists
    */
-  credentialsExist (
+  credentialsExist(
```
Comment (Aschen) [inline src/controllers/Auth.ts:240]:
```suggestion
  credentialsExist (
```

File: src/controllers/Auth.ts [hunk 11]
```diff
@@ -213,7 +213,7 @@ export class AuthController extends BaseController {
    * @returns An object representing the new credentials.
    *    The content depends on the authentication strategy
    */
-  createMyCredentials (
+  createMyCredentials(
```
Comment (Aschen) [inline src/controllers/Auth.ts:216]:
```suggestion
  createMyCredentials (
```

File: src/controllers/Auth.ts [hunk 12]
```diff
@@ -175,7 +175,7 @@ export class AuthController extends BaseController {
    *
    * @returns A token validity object
    */
-  checkToken (token?: string): Promise<{
+  checkToken(token?: string): Promise<{
```
Comment (Aschen) [inline src/controllers/Auth.ts:178]:
```suggestion
  checkToken (token?: string): Promise<{
```

File: src/controllers/Auth.ts [hunk 13]
```diff
@@ -99,7 +99,7 @@ export class AuthController extends BaseController {
    *
    * @param requestPayload Request to check
    */
-  checkRights (requestPayload: RequestPayload): Promise<boolean> {
+  checkRights(requestPayload: RequestPayload): Promise<boolean> {
```
Comment (Aschen) [inline src/controllers/Auth.ts:102]:
```suggestion
  checkRights (requestPayload: RequestPayload): Promise<boolean> {
```

File: src/controllers/Auth.ts [hunk 14]
```diff
@@ -50,10 +50,10 @@ export class AuthController extends BaseController {
    * Do not add the token for the checkToken route, to avoid getting a token error when
    * a developer simply wishes to verify their token
    */
-  authenticateRequest (request: any) {
-    if ( !this.authenticationToken
+  authenticateRequest(request: any) {
+    if (!this.authenticationToken
```
Comment (Aschen) [inline src/controllers/Auth.ts:54]:
```suggestion
  authenticateRequest (request: any) {
    if ( ! this.authenticationToken
```

File: src/controllers/Auth.ts [hunk 15]
```diff
@@ -17,7 +17,7 @@ export class AuthController extends BaseController {
    * constructor
    * @param kuzzle
    */
-  constructor (kuzzle) {
+  constructor(kuzzle) {
```
Comment (Aschen) [inline src/controllers/Auth.ts:20]:
```suggestion
  constructor (kuzzle) {
```

**Review by Aschen — APPROVED** [review]
Autolinter

**Review by Aschen — CHANGES_REQUESTED** [review]
You need to bump the package version `npm version patch`

**Review by Aschen — APPROVED** [review]

**Review by scottinet — APPROVED** [review]

**Pull request merged**